README.md

```diff
@@ -5,12 +5,43 @@
 [![pypi](https://img.shields.io/pypi/v/lamindb?color=blue&label=pypi%20package)](https://pypi.org/project/lamindb)
 [![PyPI Downloads](https://img.shields.io/pepy/dt/lamindb?logo=pypi)](https://pepy.tech/project/lamindb)
 
-# LaminDB - A data framework for biology
+# LaminDB - A data lakehouse for biology
 
-<!-- first two sentences sync from preface.md -->
+LaminDB is an open-source data lakehouse to organize biological datasets through validation & annotation providing data lineage, findability, queryability, reusability, and reproducibility.
 
-LaminDB is an open-source data framework to enable learning at scale in computational biology.
-It lets you track data transformations, validate & annotate datasets, and query a built-in database for biological metadata & data structures.
+<details>
+<summary>Why?</summary>
+
+Reproducing analytical results or understanding how a dataset or model was created can be a pain.
+Leave alone training models on historical data, LIMS & ELN systems, orthogonal assays, or datasets generated by other teams.
+
+Biological datasets are typically managed with versioned storage systems (file systems, object storage, git, dvc), GUI-focused community or SaaS platforms (LIMS & ELN systems), structureless data lakes, rigid data warehouses (SQL, monolithic arrays), and data lakehouses for tabular data.
+
+LaminDB goes further with a lakehouse that models biological datasets beyond tables with enough structure to enable queries and enough freedom to keep the pace of R&D high.
+
+For data structures like `DataFrame`, `AnnData`, `.zarr`, `.tiledbsoma`, etc., LaminDB tracks the rich context of biological research and uses it to validate and index datasets to enable queries. In particular, you get
+
+- data lineage: trace data sources and code; scientists and machine learning models
+- domain knowledge and experimental metadata: the features and labels derived underlying models and analyses
+</details>
+
+**Highlights.**
+
+- **data lineage:** track inputs & outputs of notebooks, scripts, functions & pipelines with a single line of code
+- **unified access:** query any number of storage locations (local, S3, GCP, https, HF, R2), SQL databases (Postgres, SQLite) & ontologies
+- **biology aware:**
+  - _data structures:_ validate, annotate & query formats like `DataFrame`, `AnnData`, `MuData`, ... backed by `parquet`, `zarr`, HDF5, LanceDB, DuckDB, ...
+  - _entities:_ organize features & labels in registries for experimental metadata ([LIMS](https://en.wikipedia.org/wiki/Laboratory_information_management_system)) & extendable ontologies
+- **queryable & batch-loadable:** manage schemas to impose integrity on arrays for distributed queries & batch loading
```

Comment:
This looks a bit strange to me, because here the section is about querying and batch loading, but it starts with schemas, a bit confusing to me.

Reply:
I don't find an easier way to put it. lamindb gives you schemas to enforce integrity which enables the distributed access (+ also gives nice abstractions that wrap polars/pyarrow/anndata etc.)

Reply:
this still reads a bit strange to me,

maybe "perform distributed queries and batch loading, with schemas ensuring array integrity"?